sunbeam-python/sunbeam/clusterd/cluster.py

```diff
@@ -243,7 +243,13 @@ def get_status(self) -> dict[str, dict]:
         """Get status of the cluster."""
         cluster = self._get("/1.0/status")
         members = cluster.get("metadata", {})
-        return {member["name"]: {"status": member["status"]} for member in members}
+        return {
+            member["name"]: {
+                "status": member["status"],
+                "address": member["address"].split(":")[0],
```

Comment:
This will fail in case of an IPv6 deployment

Reply:
Changed the logic to pass address in format of <ip>:<port> and handled the ip matching in the LocalClusterStatusStep